artifact-app/build.rs

```diff
@@ -53,8 +55,22 @@ fn check_deps() {
 
 fn build_mdbook() {
     println!("Building the book");
+
+    let mdbook_dir = if cfg!(windows) {
+
+        let regex = Regex::new("[a-zA-Z].+").unwrap();
+        let shortened = regex.find(BOOK.as_path().to_str().expect("Invalid book dir"))
```

Comment:
`BOOK.as_path().to_str().expect("Invalid book dir")` is being done twice and makes the code harder to read.

Reply:
Yep, that was pretty ugly stuff I left in from just trying to get it to work - sorry for letting it sneak into the PR. Updated now with more appropriate code.